ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/integration/casework-regression/74136-case-management-page-validation.js

```diff
@@ -7,16 +7,15 @@ describe("User can manage cases from the case management page", () => {
 		cy.storeSessionData();
 	});
 
-	const searchTerm =
-		"Accrington St Christopher's Church Of England High School";
+	let term = ""

```

Comment:
is this variable used anywhere, can it be removed?

Reply:
It is used in the tests yes, I've initialised at the top for consistency, and to avoid scoping issues within it blocks.